packages/widgetbook/README.md

```diff
@@ -38,19 +52,30 @@ class HotReload extends StatelessWidget {
 }
 ```
 
-### Running the Widgetbook
-
-To run the `Widgetbook` create a new main method, e.g. in `stories/main.dart`:
+In the `stories/main.dart` run the `HotReload` widget:
 
 ```dart
 void main() {
   runApp(HotReload());
 }
 ```
 
+## Run the Widgetbook
+
+`Widgetbook` is supported on the following environments.
+
+Environment | Status | Annotation
+------------ | ------------- | ------------- 
+MacOS | ✅ | 
+Windows | ✅ |
+Web | ✅ | No hot reload, but hot restart (see [Issue 4](https://github.com/widgetbook/widgetbook/issues/4))
+iOS and Mobile | ➖ | Will run, but is not optimized. If you see a usecase for `Widgetbook` on mobile let us know. 
```

Comment:
Why iOS and Mobile? Why not iOS and Android or just Mobile?

Fun fact: Widgetbook works pretty well on Tablet :D

Reply:
woops.. should be Android indeed